eZ/Publish/Core/Base/Exceptions/ContentFieldValidationException.php

```diff
@@ -30,17 +31,27 @@ class ContentFieldValidationException extends APIContentFieldValidationException
      */
     protected $errors;
 
+    /** @var string|null */
+    protected $contentName;
+
     /**
      * Generates: Content fields did not validate.
      *
      * Also sets the given $fieldErrors to the internal property, retrievable by getFieldErrors()
      *
      * @param array<array-key, array<string, \eZ\Publish\Core\FieldType\ValidationError>> $errors
      */
-    public function __construct(array $errors)
+    public function __construct(array $errors, ?string $contentName = null)
     {
         $this->errors = $errors;
-        $this->setMessageTemplate('Content Fields did not validate');
+        $this->contentName = $contentName;
+
+        $this->setMessageTemplate('Content Fields %contentName%did not validate: %errors%');
```

Comment:
```suggestion
        $this->setMessageTemplate('Content "%contentName%" fields did not validate: %errors%');
```

Reply:
Actually, a single space has to be there in order to display message properly in case `%contentName%` is empty.